docs/general/installation/truenas.md

```diff
@@ -0,0 +1,246 @@
+
+## Installing the Jellyfin Application
+
+To install the **Jellyfin** application, go to **Apps**, click **Discover Apps**, either begin typing Jellyfin into the search field or scroll down to locate the **Jellyfin** application widget. You might need to click "Refresh Charts" if no Jellyfin app is showing in the results.
+
+![Jellyfin App Widget](/images/docs/install-truenas-1.png)
+
+Click on the widget to open the **Jellyfin** application details screen.
+
+![Jellyfin App Details Screen](/images/docs/install-truenas-2.png)
+
+Click **Install** to open the Jellyfin application configuration screen.
+
+Application configuration settings are presented in several sections, each explained below.
+To find specific fields click in the **Search Input Fields** search field, scroll down to a particular section or click on the section heading on the navigation area in the upper-right corner.
+
+![Install Jellyfin Screen](/images/docs/install-truenas-3.png)
+
+Accept the default values in **Application Name** and **Version**.
+
+Accept the defaults in **Jellyfin Configuration**, **User and Group Configuration**, and **Network Configuration** or change to suit your use case.
+You must select **Host Network** under **Network Configuration** if using [DLNA](https://jellyfin.org/docs/general/networking/dlna/).
+
+Jellyfin requires three app storage datasets.
+You can allow SCALE to create them for you, or use the dataset(s) created in [First Steps](#first-steps).
+Select the storage options you want to use for **Jellyfin Config Storage** and **Jellyfin Cache Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset or select **Host Path** to use the existing datasets created on the system.
+
+Jellyfin also requires a dataset or [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) for **Jellyfin Transcodes Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset, select **Host Path** to use an existing dataset created on the system, or select **emptyDir** to use a temporary storage volume on the disk or in memory.
+
+Mount one or more media libraries using **Additional Storage**.
+Click **Add** to enter the path(s) on your system.
+Select **Host Path (Path that already exists on the system)** or **SMB Share (Mounts a persistent volume claim to a SMB share)** in **Type**.
+Enter a **Mount Path** to be used within the Jellyfin container. For example, the local Host Path /mnt/tank/video/movies could be assigned the Mount Path /media/movies.
+Define the **Host Path** or complete the SMB Share Configuration fields.
+See [Mounting Additional Storage](#mounting-additional-storage) below for more information.
+
+Accept the defaults in **Resource Configuration** or change the CPU and memory limits to suit your use case.
```

Comment:
I think the resources configuration section at the very bottom of this page would be more suited here.